### PR TITLE
added chocolate_plaid_collection

### PR DESCRIPTION
**Pull request by RaionArt**
Chocolate_plaid_collection consists of:
-`chocolate_plaid_bow`
-`chocolate_plaid_dress`
-`golden_necklace`

Also added the topic `chocolate_preference` to have the outfit unlockable on higher affinity.

### CODE REVIEW

File: game/script-topics.rpy [hunk 1]
```diff
@@ -8758,14 +8758,38 @@ label talk_chocolate_preference:
...
-    extend 2tlrsssbl " Jeez."
-    n 1unmaj "What about you,{w=0.2} [player]?{w=0.75}{nw}"
-    n 1fbkwreex "...Wait!{w=1}{nw}"
-    extend 1fcsbg " Don't tell me!"
-    n 4fcssresp "..."
 
+    if (
+        not jn_outfits.get_outfit("jn_chocolate_plaid_collection").unlocked
+        and Natsuki.isAffectionate(higher=True)
+        and persistent.jn_custom_outfits_unlocked
```
Comment (Blizzardsev) [inline game/script-topics.rpy:8765]: Good use of conditioning here - well done!

File: game/script-topics.rpy [hunk 2]
```diff
@@ -8758,14 +8758,38 @@ label talk_chocolate_preference:
...
+    ):
+        # Unlock chocolate plaid outfit, if custom outfits unlocked
+        n 1tslsl "..."
+        n 4uwdajesu "Oh!{w=0.5}{nw}"
+        extend 2fcscs " Kinda reminds me of one of my favourite dresses,{w=0.3} acutally!"
+        n 4fllss "It's like all types of chocolates combined.{w=0.75}{nw}"
+        extend 3fchcs " Ehehe."
```
[the remark this answers is not in the document]
Comment (Blizzardsev) [reply game/script-topics.rpy:8772]: I should note: I've left the sprite codes empty so you can try giving those lines a shot if you agree with the suggestions!

**Review by Blizzardsev — CHANGES_REQUESTED** [review]
Just added some thoughts!

I think the wording could use adjustment, but the ideas behind it are solid - I like the idea of Natsuki spelling out the link between the clothes and the topic; it helps solidify why it's being unlocked specifically and feels natural!

Good work on the code side of things: you know what you're doing with the registration code, and I'm happy to see how you managed the implementation/switch in the dialogue itself: well done! We might want to consider the event that a user has already read the topic and if it should be unlocked automatically for those that have, but that's just a thought.

File: game/script-topics.rpy [hunk 3]
```diff
@@ -8758,14 +8758,38 @@ label talk_chocolate_preference:
...
+        $ jn_outfits.get_outfit("jn_chocolate_plaid_collection").unlock()
+
+    else:
+        n 1fllajlsbr "A-{w=0.2}anyway!{w=0.75}{nw}"
+        extend 1fcsajl " Enough of me yammering on again.{w=0.75}{nw}"
+        extend 2tlrsssbl " Jeez."
+        n 1unmaj "What about you,{w=0.2} [player]?{w=0.75}{nw}"
+        n 1fbkwreex "...Wait!{w=1}{nw}"
+        extend 1fcsbg " Don't tell me!"
+        n 4fcssresp "..."
```
[the remark this answers is not in the document]
Comment (RaionArt) [reply game/script-topics.rpy:8790]: Ah! Tbh I wanted to do it this way, but I wasn't sure if it works without an else, this is fine by me

**Review by Blizzardsev — APPROVED** [review]

**Pull request merged**